modules/olm-updating-index-image.adoc

```diff
@@ -34,21 +34,42 @@ endif::[]
 [source,terminal]
 ----
 $ opm index add \
-    --bundles <registry>/<namespace>/<new_bundle_image>@sha256:<digest> \//<1>
-    --from-index <registry>/<namespace>/<existing_index_image>:<tag> \//<2>
-    --tag <registry>/<namespace>/<existing_index_image>:<tag> \//<3>
-    --pull-tool podman <4>
+    --bundles <bundle_registry>/<bundle_namespace>/<new_bundle_image>@sha256:<digest> \
+    --from-index <mirror_registry>/<mirror_namespace>/<existing_index_image>:<existing_tag> \
+    --tag <mirror_registry>/<mirror_namespace>/<existing_index_image>:<updated_tag> \
+    --pull-tool podman
+----
++
+where:
++
+[small]
+--
+<bundle_registry>:: Specifies the hostname of the registry where the bundle is stored, such as `quay.io`.
+<bundle_namespace>:: Specifies the namespace of the bundle registry, such as `ocs-dev`.
+<new_bundle_image>:: Specifies the new bundle image you want to add to the mirror registry, such as `ocs-operator`.
+<digest>:: Specifies a comma-separated list of additional bundle images to add to the index.
+<mirror_registry>:: Specifies the hostname of your mirror registry, such as `quay.io` or `mirror.example.com`.
+<mirror_namespace>:: Specifies the namespace of your mirror registry, such as `abc`.
+<existing_index_image>:: Specifies the existing index that was previously pushed, such as `abc-redhat-operator-index`.
+<existing_tag>:: Specifies an existing image tag that was previously pushed, such as `pass:a[{product-version}]`.
+<updated_tag>:: Specifies the image tag that you want the updated index image to have, such as `pass:a[{product-version}].1`.
+--
++
+.Example command
+[source,terminal,subs="attributes+"]
+----
+$ opm index add \
+    --bundles quay.io/ocs-dev/ocs-operator@sha256:<digest> \
```

Comment:
Might as well use a random digest here, since it's an example anyway (even if we use an existing real digest, the image's existence doesn't have to be a prereq for this doc to be valid)